### PR TITLE
feat: redo home landing page for new users

### PR DESCRIPTION
**Pull request by cabreraalex**
# Description
Fix ZEN-244

Also adds a screen if a project si selected on mobile.

Screenshot 2023-10-18 at 21 11 25
Screenshot 2023-10-18 at 21 11 15
Screenshot 2023-10-18 at 21 27 01

### CODE REVIEW

File: frontend/src/routes/(app)/(home)/reports/+page.svelte [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	export let data;
 </script>
 
-<div class="flex flex-wrap mb-6 overflow-y-scroll h-full content-start">
```
Comment (Sparkier) [inline frontend/src/routes/(app)/(home)/reports/+page.svelte:7]: Are these still scrolling? They should, right?

File: frontend/src/routes/(app)/project/[owner]/[project]/[page=exploreOrCompare]/+page.svelte [hunk 2]
```diff
@@ -1,11 +1,27 @@
...
 </script>
 
-<MetadataPanel compare={data.compare} />
-<div class="mx-5 flex flex-col flex-grow w-1">
-	<InstanceView compare={data.compare} />
-</div>
+<svelte:window bind:innerWidth={width} />
+
+{#if width < 600}
+	<div class="text-lg p-10 m-10 rounded-md bg-primary text-white">
```
Comment (Sparkier) [inline frontend/src/routes/(app)/project/[owner]/[project]/[page=exploreOrCompare]/+page.svelte:15]: Nit: Looks pretty dark on your screenshot. Maybe primary-light?

**Review by Sparkier — APPROVED** [review]
Love it.

File: frontend/src/routes/(app)/(home)/reports/+page.svelte (hunk 1 above)
Comment (cabreraalex) [reply frontend/src/routes/(app)/(home)/reports/+page.svelte:7]: Yup it scrolls the whole panel now, better on mobile and works on desktop

Comment (Sparkier) [reply frontend/src/routes/(app)/(home)/reports/+page.svelte:7]: Looking at in on hub I think this is a regression. The whole page now scrolls which makes the header go away. I think we should revert this.

**Pull request merged**